etc/profile-m-z/mupdf.profile

```diff
@@ -41,3 +42,6 @@ private-tmp
 
 dbus-user none
 dbus-system none
+
+memory-deny-write-execute
```

Comment:
Sure that this does not break `mupdf-gl`?

Reply:
I've checked several times and both mupdf and mupdf-gl both work with mdwe. At least they do on Arch Linux...